Iron-code/node-todo-api
# Setup linting in the project

#### Checklist:


- [ ] All test are passing. 
- [ ] Current changes in the pr have the necessary test.
- [ ] Api Documentation follows api standards.
- [ ] Your commit messages format follows our commit message guideline.


#### What does this pr do?
adds linting to project

#### How should this be manually tested?
run `npm install`
1. edit `src/app.js` by adding some spaces
2. run `npm run  lint`
3. notice the file is linted and the spaces you added removed



#### What are the relevant issues?
#6
